code/go/0chain.net/smartcontract/storagesc/stakepool.go

```diff
@@ -538,6 +563,8 @@ func (ssc *StorageSmartContract) stakePoolUnlock(
 		return "", common.NewErrorf("stake_pool_unlock_failed",
 			"stake pool staking error: %v", err)
 	}
+	//todo merge problem
+	//sp.emitSaveEvent(spr.ProviderType, spr.ProviderID, balances)
```

Comment:
Should we create an issue to track this TODO?

Reply:
no,it is updated inside `sp.stakeForProvider(...)`